tests/console/java.pm

```diff
@@ -37,20 +38,37 @@ sub run {
     # https://www.suse.com/releasenotes/x86_64/SUSE-SLES/15-SP2/#development-java-versions
     # java-11-openjdk                   -> Basesystem
     # java-10-openjdk & java-1_8_0-ibm  -> Legacy
-    my $cmd = 'install --auto-agree-with-licenses ';
-    $cmd .= (is_sle('15+') || is_leap) ? 'java-11-openjdk* java-1_*' : 'java-*';
```

Comment:
Is this not necessary anymore?

Reply:
for tw we need to install java-*, but I don't want change this part atm becasue I am not sure about sle or leap(maybe this is needed in other java tests?)

Reply:
It seems like installing java 10 and java 11 on SLE fails: https://bugzilla.suse.com/show_bug.cgi?id=1137466

Reply:
this issue should be resolved extra for SLE, not in this PR for java test on transactional server.

Reply:
SLE and Leap also have a transactional server pattern and just keeping the condition for the package list there wouldn't even make the code more complex

Reply:
good, agreed on this. Please see changes and verification runs:

http://10.162.23.47/tests/8526#step/java/35 (transactional server)
http://10.162.23.47/tests/8527#step/java/24 (sle 15 sp3 extra_test_textmode)

Reply:
This is not used in the `transactional-update` case